spyderlib/plugins/runconfig.py

```diff
@@ -467,8 +474,15 @@ def setup_page(self):
         interpreter_layout.addWidget(self.dedicated_radio)
         interpreter_layout.addWidget(self.systerm_radio)
         
-        wdir_group = QGroupBox(_("Working directory"))
-        wdir_bg = QButtonGroup(wdir_group)
+        general_group = QGroupBox("General settings")
+        command_options_enabled = self.create_checkbox(_("Command line options:"),

```

Comment:
I also would agree on this one with @ccordoba12, it is a setting that has no real meaning as a global setting